examples/makefile

```diff
@@ -18,53 +18,98 @@ CASES			= ${TRAINING_CASES} ${TESTING_CASES}
 
 setup: dir mesh plot_config
 
+# Create the directory structure
+# ==============================
+#
+#   $(MODEL)
+#      ├── data
+#      ├── outputs
+#      │     └── $(TESTING_CASES)
+#      └── plots
 dir:
 	mkdir -p $(MODEL)/data
 	mkdir -p $(MODEL)/outputs
 	mkdir -p $(MODEL)/plots
-	touch $(MODEL)/__init__.py
-	for test_case in $(TESTING_CASES); do \
-		mkdir -p $(MODEL)/outputs/$$test_case; \
+	for case in $(TESTING_CASES); do \
+		mkdir -p $(MODEL)/outputs/$$case; \
 	done
 
+# Generate meshes
+# ===============
+#
+# Meshes are generated for all training and testing cases.
+#  * First, a gmsh geometry file is generated using the
+#    `meshgen.py` script.
+#  * Then the geometry files are used to construct meshes
+#    in the .msh format.
+#
+# Gmsh is set to use the "pack" algorithm, which means that
+# the initial meshes are quasi-uniform. That is, they are as
+# close to uniform as they can be, given that the turbines
+# are to be explicitly meshed.
 mesh:
 	touch timing.log
 	d=$$(date +%s) && \
-	for test_case in $(CASES); do \
-		python3 meshgen.py $(MODEL) $$test_case; \
-		gmsh -2 -algo pack $(MODEL)/meshes/$$test_case.geo -o $(MODEL)/meshes/$$test_case.msh; \
+	for case in $(CASES); do \
+		python3 meshgen.py $(MODEL) $$case; \
+		gmsh -2 -algo pack $(MODEL)/meshes/$$case.geo -o $(MODEL)/meshes/$$case.msh; \
 	done && \
 	date >> timing.log && \
 	git log -n 1 --oneline >> timing.log && \
 	echo "Meshes built in $$(($$(date +%s)-d)) seconds" >> timing.log
 
+# Plot configurations
+# ===================
+#
+# Plot the configurations for a subset of the training cases
+# and all of the testing cases. The domain geometry and turbine
+# locations are shown, along with the physical parameters used.
```

Comment:
Technically not all of the testing cases, but those that are listed in `$(MODEL)/config.py`.